src/metric/gedLib/Application.cpp

```diff
@@ -0,0 +1,1638 @@
+#include "Application.h"

```

Comment:
Is this an external library? If and library cannot be brought in
via the git sub-module mechanism, all we can do is mention it in the README and tell users how to get it and where to put it. We just cannot store other peoples library in the repo.